### ISSUE TITLE
UnusualTaunts legacy issues

### ISSUE THREAD

**Issue by q1085909155** [NoyWasTaken/TF2_VIP#14]
I added the latest TF2 update mocking effects to this plugin myself. However, some effects will remain on the map and will only disappear when the player reenters the server
![屏幕截图(1)](https://github.com/NoyWasTaken/TF2_VIP/assets/69096633/2c17d5e3-0d3a-4b8a-8537-dae6c8810cf4)

**Comment by NoyWasTaken** [comment]
Hey,
Could you post here the changes that you've made so I can test it out?

**Comment by q1085909155** [comment]
Of course, I've made some language changes, and there have been a few bug fixes that prevent the server from crashing when replacing UnusualTaunt during taunting
[VIP_UnusualTaunts.zip](https://github.com/NoyWasTaken/TF2_VIP/files/13326327/VIP_UnusualTaunts.zip)

**Comment by ShamHyper** [comment]
![image](https://github.com/NoyWasTaken/TF2_VIP/assets/104412752/981052b6-2b77-46d7-94b9-c13510d8aa3a)
Same problem. Need to be fixed :)

**Comment by NoyWasTaken** [comment]
It seems like the issue is with "complicated" unusuals.
I will try and find time to work on a fix.

**Comment by NoyWasTaken** [comment]
I've looked into it.
Currently, the way that the unusual taunt works is by creating a particle entity and then at the end of the taunt, killing it.
For some of the effects, killing the entity leaves those "left overs".
I've looked into it and we can fix this issue by creating the unusual effect using unusual attribute (same way the unusual effect is applied to hats on VIP_Unusual).

I even found a post on AlliedModders of someone who took the VIP_UnusualTaunts plugin and did it: https://forums.alliedmods.net/showpost.php?p=2777538&postcount=13.
I will look at this code and if it works good I might use it.

Also, I will refactor the plugin so the list of unusual effects won't be hard-coded and add support for effects that need to be refire every X seconds.
